contents/tutorials/ruby-ab-tests.md

```diff
@@ -0,0 +1,278 @@
+---
+title: How to set up A/B tests in Ruby on Rails
+date: 2024-02-13
+author: ["lior-neu-ner"]
+tags: ['experimentation']
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import EventsInPostHogLight from '../images/tutorials/ruby-ab-tests/events-light.png'
+import EventsInPostHogDark from '../images/tutorials/ruby-ab-tests/events-dark.png'
+import TestSetupLight from '../images/tutorials/ruby-ab-tests/experiment-setup-light.png'
+import TestSetupDark from '../images/tutorials/ruby-ab-tests/experiment-setup-dark.png'
+
+A/B tests help you improve your Ruby on Rails app by enabling you to compare the impact of changes on key metrics. 
+
+To show you how to set one up on both the client and server side, we create a basic Rails app, add PostHog, create an A/B test, and implement the code for it.
+
+## 1. Create a Rails app
+
+First, ensure [Ruby](https://www.ruby-lang.org/en/documentation/installation/) and [Rails](https://guides.rubyonrails.org/v5.0/getting_started.html) are both installed. Then, create a new Rails app:
```

Comment:
This process can be a bit frustrating so might be helpful to help a bit more.
```suggestion
First, ensure [Ruby](https://www.ruby-lang.org/en/documentation/installation/) and [Rails](https://guides.rubyonrails.org/v5.0/getting_started.html) are both installed ([rbenv](https://github.com/rbenv/rbenv) is a useful tool for using a compatible Ruby version). Then, create a new Rails app:
```

Reply:
I dont think we need to do this. People who are reading this post are likely well-versed in Ruby and already set up,